### ISSUE TITLE
Proposal: New approach of updating translation

### ISSUE THREAD

**Issue by belldandyxtq** [optuna/doc-optuna-zh-cn#11]
## Background and Motivations
1. Too many autogenerated changes (10k+), difficult to find translation update to review among them
2. We may want to break the whole update into several PRs for parallel translation and review, but we may loose track of which file need to be translated.
3. We want to prevent showing users the WIP version, while we have only one branch (master) and the website shows the latest master.

## Proposal:
1. Create a WIP branch from master with the name of the new version (e.g. branch-2.7), and all the temporary PRs will be created against this branch.
2. Create a PR for all the autogenerated files; We only quickly check if there are any weird stuff included or not.
3. Create a issue/list of files include translation update (with check box)
4. Create PRs for each file/several files
5. After all the translations PRs are merged, create a PR from the WIP to master

What do you think

**Comment by chenghuzi** [comment]
Agreed. Now the pace of optuna development has slowed down, we will have more time to work on different parts while still keeping track of the current version. Currently we've finished the translation of v2.7 (but not the auto-generated parts). So maybe we can add a tag and then Readthedoc could generate a version-specific doc for us.

**Comment by chenghuzi** [comment]
API Reference translation progress:

- [x] optuna
- [x] optuna.cli
- [x] optuna.distributions
- [x] optuna.exceptions
- [ ] optuna.importance
- [ ] optuna.integration
  - [x] AllenNLP
  - [x] BoTorch
- [x] optuna.logging
- [ ] optuna.multi_objective
- [x] optuna.pruners
- [ ] optuna.samplers
- [x] optuna.storages
- [x] optuna.structs
- [x] optuna.study
- [ ] optuna.trial
- [ ] optuna.visualization
